src/main/java/core/basesyntax/DataField.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class DataField {
+
+    private final int workingHour;
+    private final int incomePerHour;
+    private final String date;
+    private final String employeeName;
+
+    public DataField(String data) {
+        String[] datas = data.split(" ");
+        date = datas[0];
+        employeeName = datas[1];
+        workingHour = Integer.parseInt(datas[2]);
+        incomePerHour = Integer.parseInt(datas[3]);
+
+    }
```

Comment:
Better to move this logic to separate method in `SalaryInfo` and create here constructor with params. `DataField` entity shouldn't be tied on specific string.

Reply:
++